code/weapon/weapon.h

```diff
@@ -387,6 +387,7 @@ struct weapon_info
 	gamesnd_id	impact_snd;
 	gamesnd_id disarmed_impact_snd;
 	gamesnd_id	flyby_snd;							//	whizz-by sound, transmitted through weapon's portable atmosphere.
```

Comment:
We should fix this comment.  Otherwise, hurray for simple PRs!

Reply:
Fix what? It's completely correct, aside from the obvious tongue-in-cheek joke

Reply:
Oh sorry, misread your PR description then.  What do you mean by: "plays while the player has one of these weapons in flight"?  It plays if the player fires it until that weapon dies?

Reply:
Yes, if the player fires it, it just plays until the weapon dies. It's a little clearer in the code where it is called `hud_in_flight_snd` (and grouped with the more obviously HUD sounds `hud_tracking_snd` and `hud_locked_snd`), but I plan on updating the wiki to clear up this possible confusion.

Reply:
Ok, that's what I'm saying should be added to that comment: that it's for player weapons only.